doc/source/whatsnew/v0.21.0.txt

```diff
@@ -439,12 +439,35 @@ Other API Changes
 
 Deprecations
 ~~~~~~~~~~~~
-- :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
 
+- :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
 - ``pd.options.html.border`` has been deprecated in favor of ``pd.options.display.html.border`` (:issue:`15793`).
 
 - :func:`SeriesGroupBy.nth` has deprecated ``True`` in favor of ``'all'`` for its kwarg ``dropna`` (:issue:`11038`).
 
+Series.argmax and Series.argmin
```

Comment:
add a ref to this subsection. make the title something like

``Series.argmin/max`` are deprecated.